lightgbm-sys/Cargo.toml

```diff
@@ -14,4 +14,4 @@ libc = "0.2.81"
 
 [build-dependencies]
 bindgen = "0.56.0"
-cmake = "0.1"
+cmake = "=0.1.45"
```

Comment:
@necnec I don't like to specify a minor version to avoid conflicts with other libraries. If you have any inconveniences, please write them in the comments.